app/src/views/search-engine/components/articles-list/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import {
+  Grid,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+  Card,
+  CardContent,
+  CardActions,
+  Typography,
+  Chip,
+  Button,
+} from '@material-ui/core';
+import styles from './styles';
+
+const ArticlesList = ({ articles }) => {
+  const [currentArticle, setCurrentArticle] = useState(undefined);
+  const classes = styles();
+
+  return (
+    <Grid container spacing={3}>
+      {articles.map((article) => {
+        const contentPreview =
+          article.content.length > 200
+            ? `${article.content.substring(0, 200)}...`
+            : article.content;
+
+        return (
+          <Grid item xs={12} sm={6} md={3} key={article.id}>
+            <Card className={classes.card} onClick={() => setCurrentArticle(article)}>
+              <CardContent>
+                <Typography gutterBottom variant="h5" component="h2">
+                  {`Artículo ${article.number}`}
+                </Typography>
+                <Typography
+                  variant="body1"
+                  color="textSecondary"
+                  component="p"
+                  className={classes.cardContent}
+                >
+                  {contentPreview}
+                </Typography>
+                <Typography
+                  variant="caption"
+                  color="textSecondary"
+                  component="p"
+                  alignLeft
+                  noWrap
+                  className={classes.keywords}
+                >
+                  {`Palabras clave: ${article.keywords.join(', ')}`}
```

Comment:
Are we displaying the keywords to the user?

Reply:
Of course this is just me playing around with the design to see what fits best, but if you think about it, showing the user what keywords relate to each article, may educate them in doing more efficient queries. Since it may help them understand what is the algorithm looking for depending on what they search for.